net/netbird/Makefile

```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=netbird
-PKG_VERSION:=0.27.3
+PKG_VERSION:=0.27.5
```

Comment:
```suggestion
PKG_VERSION:=0.27.6
```

Reply:
Thanks for the review, after compiling and testing I'll update the PR.

Reply:
0.27.7 has been released 🤐